backend/app/services/form_detector.py

```
@@ -198,85 +209,91 @@ def _extract_cv_value(self, cv_data: Dict[str, Any], cv_path: str, transform_fun
             if isinstance(category, dict):
-                items = category.get('items', [])
+                items = category.get("items", [])
                 if items:
-                    skill_lists.append(', '.join(items))
-    
-        return ', '.join(skill_lists)
+                    skill_lists.append(", ".join(items))
+
+        return ", ".join(skill_lists)
 
     def _format_experience(self, data):
```

Ruff (0.12.2)

Missing return type annotation for private function `_format_experience`

Add return type annotation: `str`

(ANN202)